### PR TITLE
update links in 2.4.7.md

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**

### CODE REVIEW

File: docs/2.4.7.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
 
-The [ProxySQL Admin (proxysql-admin)](https://docs.percona.com/proxysql/v2-config.html) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](https://docs.percona.com/proxysql/proxysql-v1.html) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](https://docs.percona.com/proxysql/psql-functions.html) supports Percona XtraDB Cluster without custom scripts.
+The [ProxySQL Admin (proxysql-admin)](proxysql-admin-tool-v2-config.md) utility simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](proxysql-v1.md) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](proxysql-admin-tool-functions.md) supports Percona XtraDB Cluster without custom scripts.
```
Comment (patrickbirch) [inline docs/2.4.7.md:12]: utility? tool?

File: docs/2.4.7.md [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
 
-The [Percona Scheduler Admin (percona-scheduler-admin)](https://docs.percona.com/proxysql/psa-scheduler.html) tool can automatically perform a failover due to node failures, service degradation, or maintenance. Percona Scheduler admin has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
+The [ProxySQL 2.x.x and pxc_scheduler_handler](psh-overview.md) tool can automatically perform a failover due to node failures, service degradation, or maintenance. The pxc_scheduler_handler tool has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
```
Comment (patrickbirch) [inline docs/2.4.7.md:14]: utility? tool?

Comment (alina-derkach-oaza) [reply docs/2.4.7.md:14]: fixed

File: docs/2.4.7.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/2.4.7.md]: fixed

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**